librosa/effects.py

```diff
@@ -660,3 +661,65 @@ def preemphasis(y, coef=0.97, zi=None, return_zf=False):
         return y_out, z_f
 
     return y_out
+
+
+def deemphasis(y, coef=0.95, zi=None):
+    """De-emphasize an audio signal with the inverse operation of preemphasis():
+
+    If y~ = preemphasis(y, coef=coef, zi=zi)
```

Comment:
i don't recommend that you use the notation `y~` in your pseudo-code to mean "\tilde{y}" in LaTeX. Instead, call x the input of preemphasis and y the input of deemphasis.

y = preemphasis(x, coef=coef, zi=zi)
x = deemphasis(y, coef=coef, zi=zi)

Also, i would prefer an equal sign rather than an arrow (`->`)
x[t] = y[t] + coef * y[t-1]

so that your pseudo-code looks as close as possible to actually runnable Python code

Reply:
Fixed in https://github.com/librosa/librosa/pull/1293/commits/5771637b181351a885e2fee266c0a03a397b4e09